packages/binarytree/src/binaryTree.ts

```diff
@@ -578,15 +578,7 @@ export class BinaryTree {
    * @param proof
    */
   async fromProof(_proof: Uint8Array[]): Promise<BinaryTree> {
-    const proofTrie = await createBinaryTree()
-    const putStack: [Uint8Array, BinaryNode][] = _proof.map((bytes) => {
-      const node = decodeBinaryNode(bytes)
-      return [this.merkelize(node), node]
-    })
-    await proofTrie.saveStack(putStack)
-    const root = putStack[0][0]
-    proofTrie.root(root)
-    return proofTrie
+    return binaryTreeFromProof(_proof)
```

Comment:
Do we even need these class methods anymore? I'd be inclined to just remove them from the class entirely.  @gabrocheleau what do you think?

Reply:
They don't make much sense to use.  But maybe they satisfy the expectations of a trie interface?

Reply:
There is no standard trie interface that I'm aware of. Since each of our `statemanager` implementations have distinct internals specific to its underlying `trie`, seems like just boilerplate that isn't easily tree-shakable to have it here as well as an independent method. I'd be in favor of removing.